### DISCUSSION TITLE
버전 정보 알려주기

### DISCUSSION THREAD

**Discussion by JIWON1923** [Team-ShortcutsZip/MacC-Team-HappyAnding#444] [New Feature Ideas]
이거 어때유?
원래 업데이트 사항 있을 때만 버튼 보여주게 하려고 했는데, 
이미 런치스크린에서 Alert로 업데이트 정보 알려주니, 어떻게 구성해야할지 고민이 되네요

|이미 최신버전|선택적 업데이트|
|---|---|
|<Img src = "https://github.com/DeveloperAcademy-POSTECH/MacC-Team-HappyAnding/assets/68676844/6acc02dd-5657-4a12-ad30-62df13397898" width = 250>|<img src = "https://github.com/DeveloperAcademy-POSTECH/MacC-Team-HappyAnding/assets/68676844/9d06136b-c03a-4a87-9d92-c56d1096e5c8" width = 250>

**Comment by HanGyeongjun** [comment]
지금 디자인해주신 저 화면은 어떤 컨택스트에서 표시되는 화면인가요? 앱 런치 시 화면일 것 같긴 한데 표시되는 정확한 시점이 궁금합니다!

**Reply by JIWON1923** [reply]
이번 제 태스크입니다!
설정 화면 버전 정보 보여주던 탭을 네비게이션으로 바꾸는 게 제 태스큽니다요
군대 생각 뿐이시군요? 🫡

**Reply by HanGyeongjun** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 아하... 그게 맞군요
런치스크린이라고 하셔서 제가 알고있는거랑 다른건가 했습니다..
탭도 프로필로 바꿔주시면 좋을거같아요

**Comment by HanGyeongjun** [comment]
저 화면에 현재 버전 숫자나, 오른쪽 화면이 표시되어야 할 경우에는 현재 버전과 최신 버전을 비교하는 텍스트가 들어가도 괜찮을 것 같다는 생각이 듭니당
그리고 다크모드일 때는 앱 로고가 배경하고 대비가 잘 될 것 같은데 라이트모드일때는 경계가 잘 안 보여서... 좋은 방법이 있을까요?

**Comment by JIWON1923** [comment]
|||
|---|---|
|<img src = "https://github.com/DeveloperAcademy-POSTECH/MacC-Team-HappyAnding/assets/68676844/6bc78188-63ea-4bc2-bde3-8b3665785845" width = 250>| <img src = "https://github.com/DeveloperAcademy-POSTECH/MacC-Team-HappyAnding/assets/68676844/173ed93c-7f88-4b56-944b-0ee148d5b666" width = 250>

쉐도우나 스트로크는 어떤가요?

**Reply by HanGyeongjun** [reply]
갑자기 떠올랐는데 그 위젯 설정 화면처럼 약간 플로팅 되어 있는 느낌을 줘도 좋을 것 같아요

![image](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-HappyAnding/assets/94854258/6f2fe2a3-1467-4f1c-9791-0a806d6cf1cf)

**Reply by JIWON1923** [reply]
오오 좋네요 디자인 해볼게용
